src/add-to-app/android/project-setup.md

```diff
@@ -293,6 +293,44 @@ dependencies {
   releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
 }
 ```
+</br>
+
+##### Kotlin DSL
+
+<?code-excerpt title="MyApp/app/build.gradle.kts"?>
+```gradle
+android {
+    buildTypes {
+        release {
+      ...
+        }
+        debug {
+          ...
+        }
+
+        create("profile") {
+            initWith(getByName("debug"))
+        }
+}
+
+dependencies {
+  // ...
+  debugImplementation 'com.example.flutter_module:flutter_debug:1.0'
+  releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
+}
```

Comment:
Can we add the `profileImplementation` here as well? Adding any necessary steps needed to add that implementation will keep this consistent with the instructions for Java based projects.

Reply:
Steps updated to add profileImplementation in Kotlin DSL based project

Reply:
It looks like we can resolve this, yes?